modules/javafx.graphics/src/main/java/com/sun/prism/es2/ES2Pipeline.java

```diff
@@ -72,13 +72,19 @@
 
         creator = Thread.currentThread();
 
+
         if (glFactory != null) {
             es2Enabled = glFactory.initialize(PrismSettings.class,
                     pixelFormatAttributes);
+            if (!es2Enabled && PrismSettings.verbose) {
+                System.out.println("GLFactory " + glFactory + "could not be initialized. ES2Pipeline not available.");
```

Comment:
space missing in front of "could"

Reply:
Note: I don't see what @dlemmermann posted but an _initial_ space seems to be missing 
"could not be initialized..."
-->
" could not be initialized..."